hedera-node/hedera-smart-contract-service-impl/src/test/java/com/hedera/node/app/service/contract/impl/test/utils/SynthTxnUtilsTest.java

```diff
@@ -44,6 +43,9 @@
 import org.junit.jupiter.api.Test;
 
 class SynthTxnUtilsTest {
+
+    private static final String LAZY_CREATION_MEMO = "";
```

Comment:
This static variable may not be required any more ?

Reply:
It is used in the test below. It replaces the static field that was previously imported.